packages/runtime/src/next/api-handler/config/app-router.ts

```diff
@@ -38,8 +41,8 @@ export async function config({
       }
     },
     customRoutes: async (route: string) => {
-      if (route === '/draft') {
-        return { res: await redirectDraftHandler(req, context, { apiKey }) }
+      if (route === '/redirect-draft') {
```

Comment:
Are intentionally changing the route here?

Reply:
Ah, never mind, this is the downstream of this change: https://github.com/makeswift/makeswift/pull/1069/files#diff-19f9b2611ef669911a4219dfef480818b939bbfdb19116162d7fdad0f6aaee79R61